### DISCUSSION TITLE
More preferences (preference country)

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#5862] [Ideas]
As per Rob Ransbottom's suggestion on the mailing list:

``` plain
It would be good to have a default country for forms per user preference.
```

Note that the United States is all the way down on most drop-downs to understand the importance of his request.

**Comment by einhverfr** [comment]
What other preferences do we need?

**Reply by ehuelsmann** [reply]
Quoting Chris himself:

> From discussion, default printer, default format, and default attachment for dropdown

**Comment by pbolger** [comment]
It'd also be good to be able to limit the visible countries in the country dropdowns. Many users won't ever select Afghanistan (for example) so having to go past it on the list is an unecessary distraction.

**Reply by ehuelsmann** [reply]
@pbolger I was going to react that you can reduce the list from the `System` menu, but as it turns out I'm mixing Country listing with Languages listing. The latter allows removal.

**Reply by ehuelsmann** [reply]
@ylavoie I guess the enhancement we were talking about today (reduction of the number of countries) has an issue filed already. Has that been dealt with already?
